cmd/exec/main.go

```diff
@@ -659,12 +659,6 @@ EXAMPLE: direktiv-sync event create -s script.txt -f filtername
 	     direktiv-sync event delete filternanme
 		 direktiv-sync event update -s script.txt -f filtername
 `,
-
```

Comment:
Cobra has good help information by default for nodes that have subcommands. Try it out for yourself: run `exec events`. So we don't need to define `Run` functions for these nodes, most of the time.